### ISSUE TITLE
enable project based URLs for posting data to opendap 

### ISSUE THREAD

**Issue by jasonfleming** [StormSurgeLive/asgs#1102]
`opendap_post2.sh` generally posts data to a single all encompassing directory hierarchy that includes metadata about the results. The purpose or project that the data is associated with has not been included in the path. It would be valuable to include the purpose, project, or even intended recipients in the path hierarchy.

**Comment by wwlwpd** [comment]
I believe we already have a shim for this, will check back to confirm if we do or not; and if so what it is.

**Comment by wwlwpd** [comment]
@jasonfleming - it was done in #931 using the variable, `OPENDAPADDPREFIX`.

```
commit b54cd464ceab8d297c6b4a631728cb1b42d1d832 (origin/issue-931)
Author: wwlwpd <46434714+wwlwpd@users.noreply.github.com>
Date:   Thu Aug 4 17:13:02 2022 -0500

    Added way to prepend a directory to the OpenDAP path formation.
    
    Issue 931: Defining OPENDAPADDROOT in an ASGS configuration file will
    cause an additional directory to be prepended to the data path
    created and used as the upload target, in output/opendap_post2.sh.
    Also writes "post.opendap.addroot" to run.properties; if not set,
    it gets recorded as "null".
    
    For example, adding the following to the ASGS config fle,
    
      OPENDAPADDROOT=replay-test
    
    Will ultimately make the files accessible in a url that is rooted
    at 'replay-test', e.g.,
    
      http://chg-1.oden.tacc.utexas.edu/thredds/catalog/asgs/replay-test/2022/al05/11/TX2008/ls6.tacc.utexas.edu/TX2008-replay-ls6/nhcConsensus/catalog.html
                                                             ^^^^^^^^^^^^
    
    If not set, the URL would normally look as follows,
    
      http://chg-1.oden.tacc.utexas.edu/thredds/catalog/asgs/2022/al05/11/TX2008/ls6.tacc.utexas.edu/TX2008-replay-ls6/nhcConsensus/catalog.html
                                                             ^^^^^^^^^^^^
    
    Resolves #931.
```

**Comment by wwlwpd** [comment]
Confirming `OPENDAPADDROOT=proj-specific-dir` will do the trick. Closing.
